### PR TITLE
Danila hm10 task3

### PR DESCRIPTION
**Pull request by TLittleProgrammer**

### CODE REVIEW

**Comment by Enotik-code** [comment]
lgfm

**Review by akupreychik — APPROVED** [review]
lgfm

File: homework/Danila/Unique Word Vocabulary/src/by/jrr/service/Library.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        }
+        else {
+            for (Book entry : list) {
+                if(entry.getAuthorBook().equals(parameter)) {
+                    return entry;
+                }
+            }
+        }
+        return null;
+    }
```
Comment (vladis1350) [inline homework/Danila/Unique Word Vocabulary/src/by/jrr/service/Library.java:50]: Bro, do you want to break it down into several functions?

**Review by vladis1350 — COMMENTED** [review]
Lgfm

File: homework/Danila/Unique Word Vocabulary/src/by/jrr/service/Library.java (hunk 1 above)
Comment (TLittleProgrammer) [reply homework/Danila/Unique Word Vocabulary/src/by/jrr/service/Library.java:50]: yes

Comment (TLittleProgrammer) [reply homework/Danila/Unique Word Vocabulary/src/by/jrr/service/Library.java:50]: yes

**Review by javaGuruBY — CHANGES_REQUESTED** [review]
wrong path

**Pull request closed without merging**